otwcode/otwarchive
# AO3-6596 Don't show an error on first login help

If logged out, rather than showing an error on the "first login help" page, show the usual message that the users needs to log in

## Issue

https://otwarchive.atlassian.net/browse/AO3-6596

## Purpose

When the user is logged out and tries to access /first_login_help, don't show them an error; instead redirect them to login with a message that they need to log in to access that page

## Testing Instructions

Attempt to access /first_login_help while logged out. You should see a message indicating that you should log in (rather than an error page)

## Credit

Eliah Hecht, he/him
